src/olympia/devhub/views.py

```diff
@@ -1611,7 +1621,9 @@ def submit_addon_upload(request, channel):
     if not RestrictionChecker(request=request).is_submission_allowed():
         return redirect('devhub.submit.agreement')
     channel_id = amo.CHANNEL_CHOICES_LOOKUP[channel]
-    return _submit_upload(request, None, channel_id, 'devhub.submit.source')
+    return _submit_upload(
+        request, None, channel_id, 'devhub.submit.source', include_recaptcha=True
```

Comment:
Note that themes have their own specific function (`submit_theme_upload()`, which passes `theme_specific=True` to bypass 2FA requirement) and you're not including the captcha there. Not sure if that was intentional (might need to check with Operations/Product what they want). In any case it would be good to cover it with tests.

Reply:
Discussed and decided not to include themese initially